iocore/net/quic/QUICStream.cc

```diff
@@ -23,30 +23,12 @@
 
 #include "QUICStream.h"
 
-#include "I_Event.h"
-#include "P_VConnection.h"
 #include "QUICStreamManager.h"
-#include "QUICDebugNames.h"
 
-#define QUICStreamDebug(fmt, ...)                                                                        \
-  Debug("quic_stream", "[%s] [%" PRIu64 "] [%s] " fmt, this->_connection_info->cids().data(), this->_id, \
-        QUICDebugNames::stream_state(this->_state.get()), ##__VA_ARGS__)
+constexpr uint32_t MAX_STREAM_FRAME_OVERHEAD = 24;
+constexpr uint32_t MAX_CRYPTO_FRAME_OVERHEAD = 16;
```

Comment:
Should be moved to `QUICCryptoStream.cc` ?

Reply:
Will do.